vmdb/app/controllers/api_controller.rb

```diff
@@ -65,8 +65,9 @@ class UnsupportedMediaTypeError < StandardError; end
   # Converted to @attr_<type> hashes at init, much faster access.
   #
   ATTR_TYPES = {
-    :time => %w(expires_on),
-    :url  => %w(href)
+    :time      => %w(expires_on),
+    :url       => %w(href),
+    :encrypted => %w(password root_password sysprep_password sysprep_domain_password)
```

Comment:
And for this, I feel we should leverage the encrypted_columns method on the class in question.  Doesn't make sense to hardcode here something that we will have to remember to keep up to date.